components/status/StatusActions.vue

```diff
@@ -112,5 +112,8 @@ function reply() {
         @click="toggleBookmark()"
       />
     </div>
+
+    <!-- ⚠️ Placeholder for UnoCSS parse. ⚠️ -->
+    <!-- <template elk-group-hover="bg-yellow/10" elk-group-hover="bg-rose/10"></template> -->
```

Comment:
cannot be moved to the script setup?

Reply:
Anywhere, but need to meet the support for `presetAttributify`.

I mean, it's just a comment node, seems to have no other effect.